test/spec.yml

```diff
@@ -417,6 +418,140 @@
+        out: |-
+            Deployed function to *
+              (use "bn invoke doomedtofailpy2" to invoke the function)
+    -   in: bn invoke doomedtofailpy2
+        err: |-
+            global name 'sdasda' is not defined
+              File "/opt/binaris/py2/main.py", line 160, in run_handler
+                response = yield gen.maybe_future(self._handler(body, request))
+            ,  File "/code/function.py", line 2, in handler
+                sdasda
+
+        exit: 1
+
+- test: Test bad user python3 code (bad-path)
+  cleanup:
+    - bn remove doomedtofailpy3
+  steps:
+    -   in: bn create python3 doomedtofailpy3
+        out: |-
+            Created function * in /home/dockeruser/test
+              (use "bn deploy doomedtofailpy3" to deploy the function)
+    -   in: |-
+            sed -i.bak "2i\    sdasda" function.py
+    -   in: bn deploy doomedtofailpy3
+        out: |-
+            Deployed function to *
+              (use "bn invoke doomedtofailpy3" to invoke the function)
+    -   in: bn invoke doomedtofailpy3
+        err: |-
+            name 'sdasda' is not defined
+              File "/opt/binaris/py3/main.py", line 160, in run_handler
+                response = yield gen.maybe_future(self._handler(body, request))
+            ,  File "/code/function.py", line 2, in handler
+                sdasda
+
+        exit: 1
+
+- test: Test bad user pypy2 code (bad-path)
+  cleanup:
+    - bn remove doomedtofailpypy2
+  steps:
+    -   in: bn create pypy2 doomedtofailpypy2
+        out: |-
+            Created function * in /home/dockeruser/test
+              (use "bn deploy doomedtofailpypy2" to deploy the function)
+    -   in: |-
+            sed -i.bak "2i\    sdasda" function.py
+    -   in: bn deploy doomedtofailpypy2
+        out: |-
+            Deployed function to *
+              (use "bn invoke doomedtofailpypy2" to invoke the function)
+    -   in: bn invoke doomedtofailpypy2
+        err: |-
+            global name 'sdasda' is not defined
+              File "/opt/binaris/py2/main.py", line 160, in run_handler
+                response = yield gen.maybe_future(self._handler(body, request))
+            ,  File "/code/function.py", line 2, in handler
+                sdasda
+
+        exit: 1
+
+- test: Test bad user java8 code (bad-path)
+  cleanup:
+    - bn remove doomedtofailjava8
+  steps:
+    -   in: bn create java8 doomedtofailjava8
+        out: |-
+            Created function * in /home/dockeruser/test
+              (use "bn deploy doomedtofailjava8" to deploy the function)
+    -   in: |-
+            sed -i.bak '6s/.*/        throw new java.lang.Error("this is very bad");/' src/main/java/ExampleFunction.java
+    -   in: gradle jar
+    -   in: bn deploy doomedtofailjava8
+        out: |-
+            Deployed function to *
+              (use "bn invoke doomedtofailjava8" to invoke the function)
+    -   in: bn invoke doomedtofailjava8
+        err: |-
+            <html>
```

Comment:
This is ugly. Please create an issue for it.
I don't want to delay this PR, but it must be handled.

Reply:
I assume you mean our bolt response is ugly correct?